Directory.Build.props

```diff
@@ -88,6 +87,7 @@
       https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json;
       https://dotnet.myget.org/F/dotnet-core/api/v3/index.json;
       https://api.nuget.org/v3/index.json;
+      https://www.myget.org/F/xunit/api/v3/index.json;
```

Comment:
This is temporary till xunit bugfixes `2.4.1` are pushed to myget: https://github.com/xunit/xunit/pull/1791#issuecomment-414497125

Reply:
We probably need something tracking removing it somehow lest it cause some subtle problem in the future.

Reply:
https://github.com/dotnet/corefx/issues/31873